install/php-ini.php

```diff
@@ -19,6 +19,7 @@
 $db_password = "{DBPWD}";
 $prefisso_tabelle = "{DBPREFIX}";
 
+define("__URL__","http://foo.com/");//TODO: mettere url nella configurazione
```

Comment:
Ciao @vittodevit!

Domanda. In LAMPSchool c'è già da qualche parte una configurazione per sapere l'URL pubblico con tanto di cartella?

Se no, suggeriamo a Marco di provare ad aggiungere in fase di installazione?

Grazie per il consulto!

Reply:
@valerio-bozzolan 
No, al momento l'url pubblico anche per la protezione degli header viene calcolato al volo dalla variabile $_SERVER
Anche io suggerisco l'aggiunta in fase di install.